docs/source/fx.rst

```diff
@@ -298,6 +290,245 @@ Miscellanea
    -  Annotations on local names within a function are not currently
       supported.
 
+Writing Transformations
+-----------------------
+
+TODO
```

Comment:
I think the rebase got messed up. The order of the sections should be 1) writing transformations 2) debugging transformations 3) limitations of symbolic tracing